pkg/sql/importer/import_job.go

```diff
@@ -1502,22 +1503,42 @@ func (r *importResumer) dropTables(
 		// writes, so even if GC has run it would not have GC'ed any keys to which
 		// we need to revert, so we can safely ignore the target-time GC check.
 		const ignoreGC = true
-		if err := sql.RevertTables(ctx, txn.DB(), execCfg, revert, ts, ignoreGC, sql.RevertTableDefaultBatchSize); err != nil {
+		if err := sql.RevertTables(ctx, txn.DB(), execCfg, []catalog.TableDescriptor{intoTable}, ts, ignoreGC,
+			sql.RevertTableDefaultBatchSize); err != nil {
 			return errors.Wrap(err, "rolling back partially completed IMPORT")
 		}
 	}
 
-	for i := range empty {
-		// Set a DropTime on the table descriptor to differentiate it from an
-		// older-format (v1.1) descriptor. This enables ClearTableData to use a
-		// RangeClear for faster data removal, rather than removing by chunks.
-		empty[i].TableDesc().DropTime = dropTime
-		if err := gcjob.ClearTableData(
-			ctx, execCfg.DB, execCfg.DistSender, execCfg.Codec, &execCfg.Settings.SV, empty[i],
-		); err != nil {
-			return errors.Wrapf(err, "clearing data for table %d", empty[i].GetID())
-		}
+	// Set a DropTime on the table descriptor to differentiate it from an
+	// older-format (v1.1) descriptor. This enables ClearTableData to use a
+	// RangeClear for faster data removal, rather than removing by chunks.
+	intoTable.TableDesc().DropTime = int64(1)
+	if err := gcjob.ClearTableData(
```

Comment:
hmm wasn't this previously conditional on the table actually being empty? What if we have a table that is `!empty`, I don't think we want to `ClearRange` it right?

Reply:
oooof. Good catch. Done.